tests/integration/cluster-users/sar-test.sh

```diff
@@ -50,6 +59,70 @@ function deleteTestBindingsRetry() {
 	for i in $(seq 1 "${MAX_RETRIES}"); do __deleteTestBindings "${MSG}" && break || __retry "${i}" || __failRetry "${MSG}" ; done
 }
 
+# Retries on errors. Note it is not "clever" and retries even on obvious non-retryable errors.
+function deleteTestNamespaceRetry() {
+	local MSG="Delete test Namespace created by Namespace Admin"
+	for i in $(seq 1 "${MAX_RETRIES}"); do __deleteTestNamespace "${MSG}" && break || __retry "${i}" || __failRetry "${MSG}" ; done
+}
+
+function createRoleBindingForNamespaceDeveloper() {
+	set +e
+	TEST=$(kubectl create rolebinding 'namespace-developer' --clusterrole='kyma-developer' --user="${DEVELOPER_EMAIL}" -n "${NAMESPACE_ADMIN_NAMESPACE}")
+	set -e
+	EXPECTED="rolebinding.rbac.authorization.k8s.io/namespace-developer created"
+
+	if [[ ${TEST} == ${EXPECTED}* ]]; then
+		echo "----> PASSED"
+		return 0
+	fi
+
+	echo "----> |FAIL| Expected: ${EXPECTED}, Actual: ${TEST}"
+
+	# If previous attempt failed (network error?), repeat just one time
+	echo "Re-trying one more time..."
+	sleep ${RETRY_TIME}
+
+	set +e
+	TEST=$(kubectl create rolebinding 'namespace-developer' --clusterrole='kyma-developer' --user="${DEVELOPER_EMAIL}" -n "${NAMESPACE_ADMIN_NAMESPACE}")
+	set -e
+	if [[ ${TEST} == ${EXPECTED}* ]]; then
+		echo "----> PASSED"
+		return 0
+	fi
+
+	echo "----> |FAIL| Expected: ${EXPECTED}, Actual: ${TEST}"
```

Comment:
Do we really need the same chunk of code twice in a row?

Reply:
please check if it is better in a current form